CONTENT/program.markdown

```diff
@@ -34,9 +34,9 @@ We are going to try hard to make this learning journey exciting for all future w
     [leveldb]( learningMaterials & screencasts ) &
     [regex]( learningMaterials & screencasts )
 
-We encourage you to pick your own projects. Come up with your own or join existing ones. We know, in the beginning this can be challenging, but no worries, we'll help you to explore :-)
+We also encourage all the participants to have some project they want to work on (but if they don't, no worries, we will help them find one or they will be able to join existing projects).
```

Comment:
I tried to copy the `pick a project` phrasing from cyber wizards, but the way you changed it, I kind of like  too.
Any thoughts about it?